src/plugins/displayLayout/mixins/objectlStyles-mixin.js

```diff
@@ -0,0 +1,59 @@
+import StyleRuleManager from "@/plugins/condition/StyleRuleManager";
```

Comment:
is that a typo in the file name?

Reply:
no. I think the .js extension is inferred.